1648177885232	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1648177900191	Sync.SyncScheduler	DEBUG	Going to sleep, doing a quick sync.
1648177900191	Sync.Service	DEBUG	User-Agent: Firefox/98.0.2 (Linux x86_64) FxSync/1.100.0.20220323001514.desktop
1648177900191	Sync.Service	INFO	Starting sync at 2022-03-25 08:41:40 in browser session umBloaRum1V8
1648177900191	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1648177900194	Sync.Status	INFO	Resetting Status.
1648177900194	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1648177900194	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1648180839486	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1648180839486	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1648180839488	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1648180839490	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648180839532	Sync.SyncScheduler	DEBUG	Woke from sleep.
1648180839532	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1648180839532	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1648180839533	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648180841533	Sync.Service	DEBUG	User-Agent: Firefox/98.0.2 (Linux x86_64) FxSync/1.100.0.20220323001514.desktop
1648180841533	FirefoxAccounts	INFO	Polling device commands.
1648180841533	Sync.Service	INFO	Starting sync at 2022-03-25 09:30:41 in browser session umBloaRum1V8
1648180841533	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:62:5) JS Stack trace: LockException@util.js:62:5
throwLockException@util.js:148:11
WrappedLock@util.js:162:15
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
1648180841533	Sync.Service	INFO	Cannot start sync: already syncing?
1648180841536	Services.Common.RESTRequest	DEBUG	GET request to https://api.accounts.firefox.com/v1/account/device/commands?index=0
1648180841538	Services.Common.RESTRequest	DEBUG	GET https://api.accounts.firefox.com/v1/account/device/commands?index=0 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1648180841538	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:488
1648180841538	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 624"  data: no] Stack trace: get status()@resource://services-common/rest.js:624
request()@resource://services-common/hawkclient.js:239
1648180841538	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 639"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:639
request()@resource://services-common/hawkclient.js:247
1648180841538	Hawk	DEBUG	(Response) /account/device/commands?index=0: code: null - Status text: null
1648180841538	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 671"  data: no] Stack trace: get headers()@resource://services-common/rest.js:671
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:316
request()@resource://services-common/hawkclient.js:255
1648180841538	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 671"  data: no] Stack trace: get headers()@resource://services-common/rest.js:671
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:316
request()@resource://services-common/hawkclient.js:256
1648180841539	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 639"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:639
_constructError()@resource://services-common/hawkclient.js:122
request()@resource://services-common/hawkclient.js:261
1648180841539	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 624"  data: no] Stack trace: get status()@resource://services-common/rest.js:624
_constructError()@resource://services-common/hawkclient.js:123
request()@resource://services-common/hawkclient.js:261
1648180841539	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 624"  data: no] Stack trace: get status()@resource://services-common/rest.js:624
_constructError()@resource://services-common/hawkclient.js:124
request()@resource://services-common/hawkclient.js:261
1648180841539	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 671"  data: no] Stack trace: get headers()@resource://services-common/rest.js:671
_constructError()@resource://services-common/hawkclient.js:130
request()@resource://services-common/hawkclient.js:261
1648180841539	FirefoxAccounts	ERROR	error GETing /account/device/commands?index=0: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]","message":null,"code":null,"errno":null}
1648180841539	Sync.SyncScheduler	ERROR	Fetching missed remote commands failed.: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]","message":null,"code":null,"errno":null}
1648180846094	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1648180846094	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1648180846096	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1648180846097	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648180846098	Sync.Service	DEBUG	User-Agent: Firefox/98.0.2 (Linux x86_64) FxSync/1.100.0.20220323001514.desktop
1648180846098	Sync.Service	INFO	Starting sync at 2022-03-25 09:30:46 in browser session umBloaRum1V8
1648180846098	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:62:5) JS Stack trace: LockException@util.js:62:5
throwLockException@util.js:148:11
WrappedLock@util.js:162:15
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
1648180846099	Sync.Service	INFO	Cannot start sync: already syncing?
1648180847492	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/194602547/info/collections failed: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1648180847493	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1648180847493	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1648180847496	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
